ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml

```diff
@@ -48,6 +48,75 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )build,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-wmcb
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-windows-machine-config-operator-master-e2e-wmcb
+    rerun_command: /test e2e-wmcb
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --branch=master
+        - --give-pr-author-access-to-namespace=true
+        - --org=openshift
+        - --repo=windows-machine-config-operator
+        - --resolver-address=http://ci-operator-configresolver
+        - --secret-dir=/usr/local/e2e-wmcb-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-wmcb
+        - --template=/usr/local/e2e-wmcb
+        command:
+        - ci-operator
+        env:
```

Comment:
We might want to add Cluster_Network_Type as [OVNKubernetes](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml#L27).

Reply:
+1 to what @suhanime found.

Reply:
I will remove the config and add the job manually to make this happen. I am assuming there is no way to feed the `ci-operator` generator with options to make this happen automatically.